File: CloudFlow/src/ro/cloudSoft/cloudDoc/presentation/client/shared/model/report/NotaGeneralaPeMembriiArbReportFilterModel.java
package ro.cloudSoft.cloudDoc.presentation.client.shared.model.report;

import java.util.Date;

public class NotaGeneralaPeMembriiArbReportFilterModel {

	private Date dataSedintaDeLa;
	private Date dataSedintaPanaLa;
	private Long bancaId;

	public Date getDataSedintaDeLa() {
		return dataSedintaDeLa;
	}

	public void setDataSedintaDeLa(Date dataSedintaDeLa) {
		this.dataSedintaDeLa = dataSedintaDeLa;
	}

	public Date getDataSedintaPanaLa() {
		return dataSedintaPanaLa;
	}

	public void setDataSedintaPanaLa(Date dataSedintaPanaLa) {
		this.dataSedintaPanaLa = dataSedintaPanaLa;
	}

	public Long getBancaId() {
		return bancaId;
	}

	public void setBancaId(Long bancaId) {
		this.bancaId = bancaId;
	}

}

File: CloudFlow/FrontEnd-App/src/app/shared/model/reports/nota-generala-pe-membrii-arb-report-filter.model.ts
import { JsonObject, JsonProperty } from "json2typescript";
import { JsonDateConverter } from "../../json-mapper";

@JsonObject
export class NotaGeneralaPeMembriiArbReportFilterModel {

	@JsonProperty("bancaId", Number)
    public bancaId: number = null;

	@JsonProperty("dataSedintaDeLa", JsonDateConverter)
	public dataSedintaDeLa: Date = null;

	@JsonProperty("dataSedintaPanaLa", JsonDateConverter)
    public dataSedintaPanaLa: Date = null;
    
}